package/chainConfigs.ts

```diff
@@ -1,22 +1,23 @@
 export const chainConfigs = {
-  [1]: {
-    FACTORY_OWNER: "0xDB392f4391462d60B8B4413ef72018Ab595Af9D0",
-    MINT_FEE_AMOUNT: 777000000000000,
-    MINT_FEE_RECIPIENT: "0xd1d1D4e36117aB794ec5d4c78cBD3a8904E691D0",
-  },
-  [10]: {
-    FACTORY_OWNER: "0x7A810DCd0f8d83B20212326813Db6EF7E9FD030c",
-    MINT_FEE_AMOUNT: 777000000000000,
-    MINT_FEE_RECIPIENT: "0x7A810DCd0f8d83B20212326813Db6EF7E9FD030c",
-  },
-  [420]: {
-    FACTORY_OWNER: "0x11cf5F667dC6AD4dEE58CB07e4AAc6a3fc7E1DCb",
-    MINT_FEE_AMOUNT: 777000000000000,
-    MINT_FEE_RECIPIENT: "0x11cf5F667dC6AD4dEE58CB07e4AAc6a3fc7E1DCb",
-  },
-  [5]: {
-    FACTORY_OWNER: "0xDC498668B5e6CC518fD58A2ADBF614Fd3A13D3a0",
-    MINT_FEE_AMOUNT: 10000,
-    MINT_FEE_RECIPIENT: "0x9444390c01Dd5b7249E53FAc31290F7dFF53450D",
-  },
-};
+    [1]: {
+  "FACTORY_OWNER": "0xDB392f4391462d60B8B4413ef72018Ab595Af9D0",
```

Comment:
Why is this changed?

Reply:
shouldn't have been in this PR; I've updated to remove this delta.